src/libraries/System.Linq.Parallel/tests/QueryOperators/AsEnumerableTests.cs

```diff
@@ -50,12 +50,13 @@ public static void AsEnumerable_LinqBinding(Labeled<ParallelQuery<int>> labeled,
         {
             IEnumerable<int> enumerable = labeled.Item.AsEnumerable();
 
-            // The LINQ Cast<T>() retains origin type for ParallelEnumerable  and Partitioner when unordered,
+            // The LINQ Cast<T>() retains origin type for ParallelEnumerable and Partitioner when unordered,
             // (and all when ordered, due to the extra wrapper)
             // although aliased as IEnumerable<T>, so further LINQ calls work as expected.
             // If this test starts failing, update this test, and maybe mention it in release notes.
             Assert.IsNotType<ParallelQuery<int>>(enumerable.Cast<int>());
             Assert.True(enumerable.Cast<int>() is ParallelQuery<int>);
+            Assert.True(enumerable.OfType<int>() is ParallelQuery<int>); // for non-nullable value types, OfType is equivalent to Cast
```

Comment:
Presumably PLINQ can't access `Iterator<T>` types, so what specifically about this change resulted in the change in behavior?

Reply:
> so what specifically about this change resulted in the change in behavior?

This:
https://github.com/dotnet/runtime/pull/99216/files#diff-9d518d344f6408b49ba2852e96d33eb701d71b93e5c83444731fbd3fd4c7ab25R19-R24